### PR TITLE
add client wrapper

### PR DESCRIPTION
**Pull request by thomas-nguy**
Create a client wrapper over 'StargateClient' and 'QueryClient' from cosmosjs

Before users need to manually import cosmosjs client into their project and configure them to be compatible with our chain.
This can be the source of errors and it requires extra effort to use the SDK
- Need to find a cosmosjs version which is compatible with crypto.com chain version
- Make sure to uses the methods supported (ie use broadcast instead of StargateSigningClient to send tx)
- QueryClient needs to be initialise with the module supported manually 

The wrapper encapsulate the transactions client and query client into an object and expose the functions only supported.
No need also to import extra packages.

### CODE REVIEW

File: lib/src/client/client.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        readonly txClient: StargateClient;
+
+        readonly queryClient:
+            | (QueryClient & AuthExtension & BankExtension & DistributionExtension & StakingExtension)
+            | undefined;
+
+        private constructor(tmClient: Tendermint34Client, txClient: StargateClient) {
+            this.txClient = txClient;
+            this.tmClient = tmClient;
+            this.queryClient = QueryClient.withExtensions(
```
Comment (thomas-nguy) [inline lib/src/client/client.ts:54]: they are the only supported module for now.
I did not add IBCExtension since its not active yet

**Review by cdc-Hitesh — DISMISSED** [review]

**Review by calvinaco — CHANGES_REQUESTED** [review]
Overall looks good to me. Left a small comment on the README. 

I notice you have committed the `yarn.lock, but in this porject we use `npm` so it is better to remove it and perhaps add it to `.gitgnore` list.

Thanks!

File: README.md [hunk 2]
```diff
@@ -125,6 +125,33 @@ console.log(signedTx.getHexEncoded());
 // Note that the result of signedTx.getHexEncoded() can be directly broadcasted to the network as a raw tx
 ```
 
+### 1.4. Sending transactions 📨
+
+The SDK uses cosmosjs stargate client to send transactions. For more information, check
```
[the remark this answers is not in the document]
Comment (thomas-nguy) [reply README.md]: thanks, fixed

File: README.md [hunk 3]
```diff
@@ -125,6 +125,33 @@ console.log(signedTx.getHexEncoded());
...
+```javascript
+// Imports
+const sdk = require("@crypto-com/chain-jslib");
+const cro = sdk.CroSDK({ network: sdk.CroNetwork.Testnet });
+const client = await cro.CroClient.connect();
+await client.broadcastTx(signedTx.encode().toUint8Array());
+```
+
+### 1.5. Query the chain ❓
+The SDK uses cosmosjs queryclient to query the blockchain. For more information, check
```
[the remark this answers is not in the document]
Comment (thomas-nguy) [reply README.md]: thanks, fixed

**Review by calvinaco — APPROVED** [review]

**Pull request merged**